src/presentation/http/router/noteSettings.test.ts

```diff
@@ -281,4 +281,76 @@ describe('NoteSettings API', () => {
 
     test.todo('Return 403 when user authorized, but not member of the team');
   });
+
+  describe('PATCH /note-settings/:notePublicId/invitation-hash ', () => {
+    test('Returns status 401 when the user is not authorized', async () => {
```

Comment:
you also need a test for access right, so only write-role users can revoke

Reply:
For now we don't have policy to check it. Should I write policy in this pr or just make todo ?

Reply:
create a todo test case

Reply:
Done